AK/Time.h

```diff
@@ -350,6 +350,16 @@ inline bool operator!=(T const& a, T const& b)
     return a.tv_sec != b.tv_sec || a.tv_nsec != b.tv_nsec;
 }
 
+// To use these, add a ``using namespace AK::TimeLiterals`` at block or file scope
+namespace TimeLiterals {
+
+constexpr Time operator""_ns(unsigned long long nanoseconds) { return Time::from_nanoseconds(static_cast<i64>(nanoseconds)); }
+constexpr Time operator""_us(unsigned long long microseconds) { return Time::from_microseconds(static_cast<i64>(microseconds)); }
+constexpr Time operator""_ms(unsigned long long milliseconds) { return Time::from_milliseconds(static_cast<i64>(milliseconds)); }
+constexpr Time operator""_sec(unsigned long long seconds) { return Time::from_seconds(static_cast<i64>(seconds)); }
```

Comment:
Can't we use non-underscored suffixes here? I think we discussed it and I don't consider any of these to be common enough prefixes (unlike "s" for example) elsewhere that we would need underscores.

Reply:
We could, but I don't think that just because we're allowed to use non-suffixed literals we are required to.

Personally, I find `100_ms` to be more readable than `100ms` to identify it as a fancy constant. For ones that take strings in, yes, it would be better imo to have `"100"ms` rather than `"100"_ms`

Reply:
okay, you can resolve this :^)